### ISSUE TITLE
Users should be able to download DATs metadata from Export Metadata pulldown

### ISSUE THREAD

**Issue by jggautier** [IQSS/dataverse#3696]
Users should be able to download DATS metadata from the Export Metadata pulldown in each dataset Metadata tab:

![screenshot 2017-03-15 21 53 03](https://cloud.githubusercontent.com/assets/18374574/23978375/1ad16fe2-09ca-11e7-9098-398a18be065b.png)

DATS specification and examples: https://github.com/biocaddie/WG3-MetadataSpecifications

**Comment by pdurbin** [comment]
My comment at https://github.com/IQSS/dataverse/issues/3697#issuecomment-287340551 applies to this issue as well.

**Comment by pdurbin** [comment]
@jggautier fast forward five years and we have many more export formats:

<img width="214" alt="Screen Shot 2022-10-04 at 10 01 38 PM" src="https://user-images.githubusercontent.com/21006/193963455-e5451d48-21a5-4659-89ed-06adcd302f24.png">

However, we still don't have DATS. Who wants DATS? Who or which installations are going to be excited if we implement this? I looked through issues and the mailing list and couldn't find any hits for DATS. Maybe you're right. Maybe we should code this up. I see "biocaddie" and "bio" make me think of this issue (so maybe we should work on it? I don't know):

- #8681

In short, I feel like this issue could use some champions or it should be closed with "not right now, maybe later".

**Comment by jggautier** [comment]
I wish I had added more information in this GitHub issue about why I opened this issue in the first place. Why Dataverse repositories should export metadata in the DATS standard. 

Someone taking this issue up might investigate it more, answering questions such as will exporting DATS metadata improve discoverability and interoperability? Would it help with data migration? And when would it? Are there discovery systems and other repositories that are interested and are able now or plan to be able to import or export metadata in the DATS standard?

More generally I'm not sure about closing an issue only because no one's working on it now.

**Comment by jggautier** [comment]
Reviewing articles about searching for biomedical data across repositories, I ran into [a 2017 article](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6460922/) about DataMed, "a prototype biomedical data search engine." Its goal is to help others more easily discover datasets in many data repositories and data aggregators. It's at https://datamed.org.

The team running the search engine hand picked several repositories and there's [a form](https://datamed.org/about.php) to let them know about other sources to consider indexing data from. They use DATS to index metadata of data.

**Comment by cmbz** [comment]
2024/09/09: Keeping, related to GREI harvesting and search/browse
